scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -175,12 +190,24 @@ <h3 class="subnav__page-title sd-flex-no-grow sd-empty">
                                 <li ng-if="type === 'monitoring'">
                                     <button ng-click="aggregate.edit('reorder', true)">
                                         <i class="icon-switches"></i>
-                                        &nbsp;<span>{{:: 'Rearrange Groups' | translate}}</span>
+                                        &nbsp;<span>{{ gettext('Rearrange Groups') }}</span>
                                     </button>
                                 </li>
                             </ul>
                         </div>
-            
+                    </div>
+                    <!-- Active filter tags -->
+                    <div class="search-tags" ng-if="aggregate.activeFilters.contentProfile.length > 0">
+                        <ul class="search-tags__list">
+                            <li ng-repeat="(type, keys) in aggregate.activeFilterTags">
+                            <span>{{ type | translate }}:
+                                <div ng-repeat="key in keys" class="tag-label tag-label--highlight2">
+                                    </span>{{ key.label }}
```

Comment:
the issue I can see locally is this markup - you open span, then open div, then close span, and then div continues.
you can see the wrong output if you run `grunt build` rather than `grunt server` or on [PR test instance](https://sdcpr-3271.test.superdesk.org/)